app/controllers/keywords_controller.rb

```diff
@@ -38,4 +39,8 @@ def set_csv_import_form
   def create_params
     params.require(:csv_import_form).permit(:file)
   end
+
+  def search_results
```

Comment:
how about just `keywords` or `search_keyword`

Reply:
Fixed in b81964c3346230cbcf63956cd19b4cd7556fd4f1